### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by acabal** [drgrigg/george-eliot_middlemarch#1]
- IDs of book-n.xhtml and parent sections in chapters should be "book-n" not "part-n"

- There is have a standard for elision poetry which occurs in some of the epigraphs, see the semantics manual and search for "Elision in poetry". I found them by running typogrify which changes the sections we want to switch to the existing pattern.

- In chapter 2's epigraph, the `span` tag should be an `i` tag, which should be inverted to non-italic for the epigraph.  Are there other cases where this occurs?

- For epigraphs that are play excerpts, they need z3998:drama and play formatting, which I think is in the semantics manual currently. If not it is in the draft manual for sure, see high-level-structural-patterns.html "Plays & Drama"

- Publication year missing from colophon

- Art year in colophon needs an en-dash not hyphen-minus

- Dedication appears to be missing styling. You can copy the full-page epigraph styling from the semantics manual and adapt if necessary.

**Comment by drgrigg** [comment]
> IDs of book-n.xhtml and parent sections in chapters should be "book-n" not "part-n"

Done
 
> There is have a standard for elision poetry which occurs in some of the epigraphs, see the semantics manual and search for "Elision in poetry". I found them by running typogrify which changes the sections we want to switch to the existing pattern.

I didn't know about this, thanks. I was puzzled by typogrify spacing what I had one way, only to have lint complain about it.

> In chapter 2's epigraph, the span tag should be an i tag, which should be inverted to non-italic for the epigraph. Are there other cases where this occurs?

There is only one other case where there is a passage in one language followed by a translation in English (in chapter 46).

However, there are many epigraphs which are entirely in one foreign language or another. Should these be tagged with `<i>`? (The xml:lang is in the parent blockquote).

> For epigraphs that are play excerpts, they need z3998:drama and play formatting, which I think is in the semantics manual currently. If not it is in the draft manual for sure, see high-level-structural-patterns.html "Plays & Drama"

It's not on the website, but I found it in the manual, thanks.

> Publication year missing from colophon

Fixed. Query, is the standard "between 1871 and 1872" or should it be "in 1871-72" ?

> Art year in colophon needs an en-dash not hyphen-minus

Fixed. See above question, though.

> Dedication appears to be missing styling. You can copy the full-page epigraph styling from the semantics manual and adapt if necessary.

Fixed.

**Comment by acabal** [comment]
> However, there are many epigraphs which are entirely in one foreign language or another. Should these be tagged with <i>? (The xml:lang is in the parent blockquote).

If the entire thing is in non-English then you can add xml:lang to the topmost tag (probably `<p>` since `<blockquote>` would include `<cite>` which is *not* non-English) and style `body [xml|lang]{ font-style: italic; }`. You also need to add the xml namespace declaration to the top of the CSS file otherwise it won't work.

> Fixed. Query, is the standard "between 1871 and 1872" or should it be "in 1871-72" ?

I don't think there's a standard yet, do whatever you like for now and that's something easily fixable later if we decide to do the opposite.

**Comment by drgrigg** [comment]
Just to clarify: is the result we are looking for that most epigraphs render to the reader as italic, regardless of the language? The exceptions being those epigraphs which have a passage in a foreign language followed by a translation in English, in which case you want the foreign bit not in italics and the English in italics? (This seems to me to be the wrong way around, if so).

Also, regarding the xml:lang attribute: I would suggest that in almost every case in this book, the contents of the cite tag, being the name of the author, can be considered to be the same language as the foreign text, eg a passage in French, with the cite being 'Pascal'. Pascal is a French name, so... I would therefore think the whole blockquote should get the language tag in such cases. Agree?

**Comment by acabal** [comment]
Yes. Epigraphs in section headers are all in italics. When text is italicized by default, text within that text that would otherwise get italics, gets Roman instead. If you copied and pasted the CSS from the semantics manual then this is already applied. But that doesn't cover the case where we don't use an `<i>` tag so you'll have to add CSS to cover that.

I don't agree on the lang tag applying to everything. We would not use an xml:lang tag on all instances of the word Pascal in plain text, would we? Or Don Quixote for example?

**Comment by drgrigg** [comment]
Ok, understood, thanks for clarifying. I'll try and get that done this morning.

**Comment by drgrigg** [comment]
OK, I think that's all done. These epigraphs have been more work than the whole of the rest of the text!
